stream-chat-android-compose/src/test/kotlin/io/getstream/chat/android/compose/ui/attachments/content/AttachmentsContentTest.kt

```diff
@@ -76,4 +83,11 @@ internal class AttachmentsContentTest : SnapshotTest {
             MediaAttachmentQuotedContent()
         }
     }
+
+    @Test
+    fun `file upload content`() {
+        snapshotWithDarkMode {
```

Comment:
Can you use `snapshotWithDarkModeRow` instead?
So the snapshot is gonna be smaller.
<img width="664" alt="Screenshot 2025-06-12 at 12 59 23" src="https://github.com/user-attachments/assets/b1ce41d1-52c2-4a62-b3a8-c3ad0a07791d" />